travis/lint-requirements.txt

```diff
@@ -2,4 +2,4 @@ prospector[with_pyroma]==0.12.7
 pep8==1.7.1
 pyarrow==0.12.1
 pylint==1.8.2
-rstcheck==3.2
```

Comment:
Should we revert this diff?

Reply:
Yes, I was getting some errors in the beginning while trying to run `lint.sh`, but now I tested without it and it's okay.

Reply:
yes, it is required @andrewmchen . Please see: https://travis-ci.org/mlflow/mlflow/jobs/549827401